deploy/ansible/roles/iroha-k8s/files/k8s-iroha.yaml

```diff
@@ -39,34 +59,80 @@ spec:
         env:
           - name: POSTGRES_PASSWORD
             value: mysecretpassword
+          - name: PGDATA
+            value: /var/lib/postgresql/data/pgdata
+        livenessProbe:
+          exec:
+            command:
+            - pg_isready
```

Comment:
Can't the whole command be inlined? Looks weird

Reply:
Agreed looks silly, I used a Postgres sample that had success with for other apps. I've edited the probe check be a one liner `command:`.